nibabel/openers.py

```diff
@@ -60,13 +60,38 @@ def readinto(self, buf):
             return n_read
 
 
-def _gzip_open(fileish, *args, **kwargs):
-    gzip_file = BufferedGzipFile(fileish, *args, **kwargs)
-
-    # Speedup for #209; attribute not present in in Python 3.5
-    # open gzip files with faster reads on large files using larger
-    # See https://github.com/nipy/nibabel/pull/210 for discussion
-    if hasattr(gzip_file, 'max_chunk_read'):
+def _gzip_open(fileish, mode='rb', compresslevel=9):
+
+    # is indexed_gzip present?
+    try:
+        from indexed_gzip import SafeIndexedGzipFile
+        have_indexed_gzip = True
+    except:
+        have_indexed_gzip = False
```

Comment:
Let's move this block to the top of `openers.py`.

Reply:
Sorry for the continually occurring style issues - I just discovered that my emacs was not liking the file-local variables in e.g. `arrayproxy.py`, so linting was not running.